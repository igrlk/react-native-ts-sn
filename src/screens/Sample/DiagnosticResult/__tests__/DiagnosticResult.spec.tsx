import React from 'react';
import renderer from 'react-test-renderer';
import { ApolloClient, InMemoryCache } from 'apollo-boost';
import { ApolloProvider } from '@apollo/react-hooks';
import { createHttpLink } from 'apollo-link-http';

import DiagnosticResult from '../DiagnosticResult';

describe('DiagnosticResult', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(
			<ApolloProvider
				client={
					new ApolloClient({
						link: createHttpLink({
							uri: 'some-uri ',
							fetch: jest.fn(() => Promise.resolve() as any),
						}),
						cache: new InMemoryCache(),
					})
				}
			>
				<DiagnosticResult
					navigation={ {state: { params: { sample: { uuid: 'test' } } } } as any}
				/>
			</ApolloProvider>,
		).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
