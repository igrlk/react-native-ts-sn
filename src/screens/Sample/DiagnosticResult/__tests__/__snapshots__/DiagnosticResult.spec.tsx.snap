// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DiagnosticResult match to snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        undefined,
        undefined,
        Object {},
      ]
    }
  >
    <View>
      <RCTScrollView>
        <View>
          <ActivityIndicator
            animating={true}
            color="#999999"
            hidesWhenStopped={true}
            size="small"
          />
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
